src/renate/updaters/learner.py

```diff
@@ -76,6 +83,18 @@ def __init__(
         self._rng = get_generator(self._seed)
         self.save_hyperparameters(ignore=self._ignored_hyperparameters())
 
+    def _possibly_populate_class_mask(self):
+        # Compute which logits to mask, if the model has a _num_outputs attribute. This is possible
+        # when self._model is a RenateBenchmarkingModule. It its not, the first forward prop will
+        # populate the _class_mask.
+        self._classes_in_current_task = unique_classes(self._train_dataset)
+        warnings.warn(f"{self._classes_in_current_task}")
```

Comment:
is this a debugging left-over? anyway, this should not be a `warning.warn` call. Instead, it should be output generated by `logging`. It probably also does not deserve warning level. If you want to leave it, please provide a bit more context in the logging message.

Reply:
For debugging purposes. Now removed.